app/helpers/application_helper.rb

```diff
@@ -492,10 +492,8 @@ def accessible_resource(obj, resource, order = :name, association: resource)
     list
   end
 
-  def accessible_related_resource(obj, relation, opts = {})
+  def accessible_related_resource(obj, relation, order: :name, where: nil)
     return [] if obj.blank?
-    order = opts.fetch(:order, :name)
-    where = opts.fetch(:where, nil)
```

Comment:
Should be safe to do this https://github.com/search?q=accessible_related_resource&type=Code&utf8=%E2%9C%93